src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -32,7 +32,11 @@
 
     public static final String FILEPATH_ID_PLACHEOLDER = "PLACEHOLDER_FOR_ID";
 
-    public static final String PARTICIPATION_TOPIC_ROOT = "/topic/participation/";
+    public static final String EXERCISE_TOPIC_ROOT = "/topic/exercise/";
+
+    public static final String NEW_SUBMISSION_TOPIC = "/topic/newSubmission";
+
+    public static final String NEW_RESULT_TOPIC = "/topic/newResults";
```

Comment:
Would be nice if singular and plural are consistent for `/topic/newSubmission` and `/topic/newResults`.

Reply:
I've changed it to `/topic/newSubmissions` (and removed some more duplications).